Stripe/STPImageLibrary.m

```diff
@@ -144,15 +92,64 @@ + (UIImage *)largeCardApplePayImage {
     return [self safeImageNamed:@"stp_card_form_applepay" templateifAvailable:YES];
 }
 
++ (UIImage *)safeImageNamed:(NSString *)imageName
+        templateifAvailable:(BOOL)templateIfAvailable {
```

Comment:
I didn't make this typo, I just moved this whole method from above into the private category. Should I still go through and fix all the refs to it?